lib/modules/datasource/github-release-attachments/index.spec.ts

```diff
@@ -0,0 +1,154 @@
+import { getDigest, getPkgReleases } from '..';
+import * as githubGraphql from '../../../util/github/graphql';
+import * as _hostRules from '../../../util/host-rules';
+import { GitHubReleaseAttachmentMocker } from './test';
+import { GithubReleaseAttachmentsDatasource } from '.';
+
+jest.mock('../../../util/host-rules');
+const hostRules: any = _hostRules;
```

Comment:
```suggestion
const hostRules = mocked(_hostRules);
```
no any, use our helper from test utils

Reply:
done. also done for the old `github-releases` data source